webapp/components/admin_console/policy_settings.jsx

```diff
@@ -74,46 +82,154 @@ export default class PolicySettings extends AdminSettings {
                     }
                 />
                 <DropdownSetting
-                    id='restrictPublicChannelManagement'
+                    id='restrictPublicChannelCreation'
                     values={[
                         {value: Constants.PERMISSIONS_ALL, text: Utils.localizeMessage('admin.general.policy.permissionsAll', 'All team members')},
                         {value: Constants.PERMISSIONS_TEAM_ADMIN, text: Utils.localizeMessage('admin.general.policy.permissionsAdmin', 'Team and System Admins')},
                         {value: Constants.PERMISSIONS_SYSTEM_ADMIN, text: Utils.localizeMessage('admin.general.policy.permissionsSystemAdmin', 'System Admins')}
                     ]}
                     label={
                         <FormattedMessage
+                            id='admin.general.policy.restrictPublicChannelCreationTitle'
+                            defaultMessage='Enable public channel creation for:'
+                        />
+                    }
+                    value={this.state.restrictPublicChannelCreation}
+                    onChange={this.handleChange}
+                    helpText={
+                        <FormattedHTMLMessage
```

Comment:
Why are we using `FormattedHTMLMessage` here?

1. the string does not have any HTML in it
2. remember that we are trying to move away from this component